contrib/devtools/Makefile

```diff
@@ -84,39 +82,42 @@ ifeq ($(UNAME_S),Darwin)
   PROTOC_ZIP ?= protoc-3.11.2-osx-x86_64.zip
 endif
 
-protoc:
+proto-tools: proto-tools-stamp
+proto-tools-stamp:
 	@echo "Installing protoc compiler..."
 	@(cd /tmp; \
 	curl -OL "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/${PROTOC_ZIP}"; \
-	unzip -o ${PROTOC_ZIP} -d /usr/local bin/protoc; \
-	unzip -o ${PROTOC_ZIP} -d /usr/local 'include/*'; \
+	unzip -o ${PROTOC_ZIP} -d $(PREFIX) bin/protoc; \
+	unzip -o ${PROTOC_ZIP} -d $(PREFIX) 'include/*'; \
 	rm -f ${PROTOC_ZIP})
 
-protoc-gen-gocosmos:
-	@echo "Installing protoc-gen-gocosmos..."
-	@go install github.com/regen-network/cosmos-proto/protoc-gen-gocosmos
-
-buf: protoc-gen-buf-check-breaking protoc-gen-buf-check-lint
-	@echo "Installing buf..."
-	@curl -sSL \
-    "https://github.com/bufbuild/buf/releases/download/v${BUF_VERSION}/buf-${UNAME_S}-${UNAME_M}" \
-    -o "${BIN}/buf" && \
-	chmod +x "${BIN}/buf"
-
-protoc-gen-buf-check-breaking:
 	@echo "Installing protoc-gen-buf-check-breaking..."
 	@curl -sSL \
     "https://github.com/bufbuild/buf/releases/download/v${BUF_VERSION}/protoc-gen-buf-check-breaking-${UNAME_S}-${UNAME_M}" \
     -o "${BIN}/protoc-gen-buf-check-breaking" && \
 	chmod +x "${BIN}/protoc-gen-buf-check-breaking"
 
-protoc-gen-buf-check-lint:
 	@echo "Installing protoc-gen-buf-check-lint..."
 	@curl -sSL \
     "https://github.com/bufbuild/buf/releases/download/v${BUF_VERSION}/protoc-gen-buf-check-lint-${UNAME_S}-${UNAME_M}" \
     -o "${BIN}/protoc-gen-buf-check-lint" && \
 	chmod +x "${BIN}/protoc-gen-buf-check-lint"
```

Comment:
this can all be removed, this is to use buf features with protoc but we do not have cmds for this

Reply:
Amended. Thanks